frontend/themes/gov-uk/client-js/index.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-var */
+/* eslint-disable no-var, vars-on-top, no-unused-vars */
```

Comment:
this is clientside govuk stuff, if I add the eslint requests, it doesn't work as govuk needs to be initialised before calling the govuk cookies and skip-to-main